meson.build

```diff
@@ -3746,92 +3746,101 @@ found = []
 missing = []
 
 foreach tuple : [
-        ['libcryptsetup'],
-        ['PAM'],
-        ['pwquality'],
-        ['libfdisk'],
-        ['p11kit'],
-        ['libfido2'],
-        ['tpm2'],
+        # dependencies
+        ['ACL'],
         ['AUDIT'],
-        ['IMA'],
         ['AppArmor'],
-        ['SELinux'],
+        ['IMA'],
+        ['PAM'],
         ['SECCOMP'],
+        ['SELinux'],
         ['SMACK'],
-        ['zlib'],
-        ['xz'],
-        ['zstd'],
-        ['lz4'],
-        ['bzip2'],
-        ['ACL'],
+        ['blkid'],
+        ['elfutils'],
         ['gcrypt'],
-        ['qrencode'],
-        ['microhttpd'],
         ['gnutls'],
-        ['openssl'],
+        ['libcryptsetup'],
         ['libcurl'],
-        ['idn'],
-        ['initrd'],
-        ['compat-mutable-uid-boundaries'],
-        ['nscd'],
-        ['libidn2'],
+        ['libfdisk'],
+        ['libfido2'],
         ['libidn'],
+        ['libidn2'],
         ['libiptc'],
-        ['elfutils'],
+        ['microhttpd'],
+        ['openssl'],
+        ['p11kit'],
+        ['pcre2'],
+        ['pwquality'],
+        ['qrencode'],
+        ['tpm2'],
+        ['xkbcommon'],
+
+        # compression libs
+        ['zstd'],
+        ['lz4'],
+        ['xz'],
+        ['zlib'],
+        ['bzip2'],
```

Comment:
maybe order alphabeticall within each section?

Reply:
I ordered the compressors by "usefulness" (somewhat vaguely defined, but e.g. bzip2 deserves to be last...).